### PR TITLE
Implemented Skeleton loader components

### PR DESCRIPTION
**Pull request by victorskl**
* Added filename caption to Circos plot Feature panel
* Minor typo improvement

### CODE REVIEW

File: src/containers/subjects/SubjectFeatureTable/index.tsx [hunk 1]
```diff
@@ -42,6 +43,17 @@ function SubjectFeatureTable(props: Props) {
     return <img src={item} alt={''} style={{ width: '100%', display: 'block' }} />;
   };
 
+  const caption = (item: string) => {
+    const filename = new URL(item).pathname.split('/').pop();
+    return (
+      <React.Fragment>
+        <div className='flex flex-wrap align-items-center justify-content-center'>
+          <h4 className='mb-1'>{filename}</h4>
```
Comment (williamputraintan) [inline src/containers/subjects/SubjectFeatureTable/index.tsx:51]:
```suggestion
          <h4 className='mb-1 text-overflow-ellipsis overflow-hidden'>{filename}</h4>
```

Seems that at long caption could exceed the length of the box. This should hide it out.

Screenshot 2023-03-27 at 11 40 29

**Review by williamputraintan — APPROVED** [review]

**Comment by williamputraintan** [comment]
Just minor css there, else LGTM!

File: src/containers/subjects/SubjectFeatureTable/index.tsx (hunk 1 above)
Comment (victorskl) [reply src/containers/subjects/SubjectFeatureTable/index.tsx:51]: 😢 I know!! Thought, you won't find it! 😭 _(hide & seek)_ Not so perfected there yet.

One for 2.2/Ocicat backlog css tuning; when we shall introduce multi-themes switcher. Let close one eye 🙈 for now, k? 🙏

Comment (williamputraintan) [reply src/containers/subjects/SubjectFeatureTable/index.tsx:51]: Maybe just commit the suggestion there, then additional css tuning can be added on `Ocicat`?

Comment (victorskl) [reply src/containers/subjects/SubjectFeatureTable/index.tsx:51]: Yah. Let's Ocicat! CSS tuning shall go across multiple themes, yes. 🙂

**Comment by victorskl** [comment]
Merging...

**Pull request merged**